src/components/ChatLog.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = ({entries, onLike, likedMessages = new Set()}) => {
```

Comment:
Super cool to see you're playing around with default values for props! I don't think it's necessary to have a default value for `likedMessages` though since it will always receive a value from App.